src/components/User/Dashboard.jsx

```diff
@@ -4,22 +4,13 @@ import ProfileHeader from "./Header";
 import Tag from "../Admin/Tag";
 import Col from "react-bootstrap/Col";
 import Row from "react-bootstrap/Row";
-import { Genders, Grades, Majors } from "@/data/Register";
-import Select from "../Select";
-import { Schools } from "@/data/Schools";
-import Radio from "../Radio";
-import Input from "../Input";
-import { FaPencil, FaCheck } from "react-icons/fa6";
+import { teamList } from "@/data/mock/Teams";
+import TeamInfo from "./TeamInfo";
+import UserInfo from "./UserInfo";
+import { mockUser } from "@/data/mock/User";
 
 const Dashboard = () => {
-  const [user, setUser] = useState({
-    phone: "123 123 1234",
-    grade: "undergraduate",
-    school: "samepl scool",
-    major: "nothing im useless",
-    gender: "everythijg",
-    team: "lololol",
-  });
+  const [user, setUser] = useState(mockUser);
   const [edit, setEdit] = useState(false);
   const handleEdit = () => {
```

Comment:
if the function only toggles, then we can integrate it directly instead of making a whole function for it